monai/transforms/spatial/array.py

```diff
@@ -703,6 +706,131 @@ def inverse(self, data: torch.Tensor) -> torch.Tensor:
         with flipper.trace_transform(False):
             return flipper(data)
 
+SUPPORTED_OPERATORS = {
+    "flippoint": FlipPointOp
+}
+
+#! Flip with operators
+class Flip(InvertibleTransform, LazyTransform):
+    backend = [TransformBackends.TORCH]
+
+    def __init__(
+        self,
+        spatial_axis: Sequence[int] | int | None = None,
+        lazy: bool = False,
+        operator=None,
+        *args,
+        **kwargs,
+    ) -> None:
+        LazyTransform.__init__(self, lazy=lazy)
+        self.kwargs = kwargs
+        self.spatial_axis = spatial_axis
+        self.operators: list[Operator] = []
+        for r in SUPPORTED_OPERATORS:  # set predefined operators as default
```

Comment:
Do you mean to put all kinds of operators (flip, rotate, crop, etc.) in this `SUPPORTED_OPERATORS`? `for-loop` all the operators in every transform?

Thanks.